senza/components/taupage_auto_scaling_group.py

```diff
@@ -1,33 +1,42 @@
+import json
+import re
+import sys
+import textwrap
 
 import click
 import pierone.api
-import textwrap
 import yaml
-import json
-import sys
-import re
-
 from senza.aws import resolve_referenced_resource
 from senza.components.auto_scaling_group import component_auto_scaling_group
 from senza.docker import docker_image_exists
 from senza.utils import ensure_keys
 
-
 _AWS_FN_RE = re.compile(r"('[{]{2} (.*?) [}]{2}')", re.DOTALL)
 
 
 def check_docker_image_exists(docker_image: pierone.api.DockerImage):
     if 'pierone' in docker_image.registry:
```

Comment:
this actually will not trigger for OS registry (registry.opensource.zalan.do) --- this check is meant to check whether we need authentication